File: client/src/types/Account.ts
type Account = {
  name: string;
  email: string;
  password: string;
  role: string;
  authCode: string;
  isVerified: boolean;
  isAutoVerified: boolean;
};

type ReturnAccount = {
  _id: string;
  name: string;
  email: string;
  password: string;
  role: string;
  authCode: string;
  isVerified: boolean;
  isAutoVerified: boolean;
};

type DoctorInList = {
  _id: string;
  name: string;
};

type Email = {
  subject: string;
  body: string;
  to: string;
};

export type { Account, ReturnAccount, DoctorInList, Email };

File: server/models/account.py
from pydantic import BaseModel, Field


class Account(BaseModel):
    name: str = Field(default="user")
    email: str = Field(default="email")
    password: str = Field(default="password")
    role: str = Field(default="role")
    authCode: str = Field(default="authCode")
    isVerified: bool = Field(default=False)
    isAutoVerified: bool = Field(default=False)


class AuthCode(BaseModel):
    authCode: list


class Email(BaseModel):
    subject: str
    body: str
    to: str
